bql/semantic/expression.go

```diff
@@ -81,6 +81,15 @@ func (o OP) String() string {
 	}
 }
 
+// cellFromRow retrieves the value of a binding from a given row.
+func cellFromRow(binding string, r table.Row) (*table.Cell, error) {
+	val, ok := r[binding]
+	if !ok {
+		return nil, fmt.Errorf("comparison operation requires the binding value for %q for row %v to exist", binding, r)
+	}
```

Comment:
This error seems to be out of scope for the function,
maybe return binding not found, and in the caller of this we reutn this message in case of error.

Reply:
Done, thank you Kloss.